CASMPET-5641 main : DOCS: Regenerate the 'etcd-client-cert' secret in the sysmgmt-health ns (#1732)
Cray-HPE/docs-csm: operations/kubernetes/Cert_Renewal_for_Kubernetes_and_Bare_Metal_EtcD.md

                     --save-config --dry-run=client -o yaml | kubectl apply -f -
      ```

>    1. Check that the certificate's expiration date is not expired.

Reviewer: ```suggestion
   1. Check the certificate's expiration date to verify that the certificate is not expired.
```

      NAME                               READY   STATUS      RESTARTS   AGE
      kube-etcdbackup-1652201400-czh5p   0/1     Completed   0          107s
      ```

1. Update the client certificate for `etcd-client`.

   1. Update the `etcd-client-cert` secret.

      ```bash
      ncn-m# kubectl --namespace=sysmgmt-health create secret generic etcd-client-cert 
                     --from-file=etcd-client=/etc/kubernetes/pki/apiserver-etcd-client.crt \
                     --from-file=etcd-client-key=/etc/kubernetes/pki/apiserver-etcd-client.key \
                     --from-file=etcd-ca=/etc/kubernetes/pki/etcd/ca.crt \
                     --save-config --dry-run=client -o yaml | kubectl apply -f -
      ```

>    1. Check that the certificate's expiration dates are not expired.

Reviewer: ```suggestion
   1. Check the certificates' expiration dates to verify that none of the certificate are expired.
```

      ```bash

>       ncn-m# kubectl get secret -n sysmgmt-health etcd-client-cert -o json | jq -r '.data."etcd-ca" | @base64d' | openssl x509 -noout -enddate

Reviewer: ```suggestion
         ncn-m# kubectl get secret -n sysmgmt-health etcd-client-cert -o json | jq -r '.data."etcd-ca" | @base64d' | openssl x509 -noout -enddate
```

>       ```

Reviewer: ```suggestion
         ```
```

>       Example output:

Reviewer: ```suggestion
         Example output:
```

      ```text
      notAfter=May  1 18:20:23 2032 GMT

>       ```

Reviewer: ```suggestion
         ```
```

>       ```bash

Reviewer: ````suggestion
      1. Check the `etcd-client` expiration date.

         ```bash
````

>       ncn-m# kubectl get secret -n sysmgmt-health etcd-client-cert -o json | jq -r '.data."etcd-client" | @base64d' | openssl x509 -noout -enddate

Reviewer: ```suggestion
         ncn-m# kubectl get secret -n sysmgmt-health etcd-client-cert -o json | jq -r '.data."etcd-client" | @base64d' | openssl x509 -noout -enddate
```

>       ```

Reviewer: ```suggestion
         ```
```

>       Example output:

Reviewer: ```suggestion
         Example output:
```

>       ```text

Reviewer: ```suggestion
         ```text
```

>       notAfter=May  4 18:20:24 2023 GMT

Reviewer: ```suggestion
         notAfter=May  4 18:20:24 2023 GMT
```

>       ```

Reviewer: ```suggestion
         ```
```

>    1. Restart Prometheus

Reviewer: ```suggestion
   1. Restart Prometheus.
```
